### ISSUE TITLE
Comparison with reactive-table

### ISSUE THREAD

**Issue by dandv** [Sinova/LiveTable#1]
How does this new package compare to the existing prior art, namely @aslagle's [reactive-table](https://atmospherejs.com/aslagle/reactive-table)?

**Comment by ShimShamSam** [comment]
The biggest difference is reactive-table is heavily tied to the server and its collections. On the other hande, LiveTable is meant to handle any array or cursor, which means your record function can do anything it needs to. The trade-off is that it's up to you to defined your publish/subscribe functions and handle sorting/filtering (LiveTable passes the current sort array and filters to your callback, so you can pass them to the subscription if needed).

In simpler terms, LiveTable is just a table that displays an array of data with nice helper features. A lot of the legwork is left to you.

I should also mention LiveTable is in development. I guess I should hide the package on Atmosphere for now.
